README.md

```diff
@@ -92,6 +92,8 @@ catch(Exception $e)
 	\QuantumPHP::add('test','warning',$e);
 }
 QuantumPHP::send();
+
+?>
```

Comment:
In a `php` file there's no need for the closing tag ! See https://www.php-fig.org/psr/psr-2/#22-files.